### DISCUSSION TITLE
Project Example

### DISCUSSION THREAD

**Discussion by aldys33** [lariat-js/playwright#51] [General]
Hi @mskelton,
do you have Playwright project example that I can use as reference to implement Lariat? Because I'm still not 100% understand when reading the docs, especially the nested collection part

**Comment by mskelton** [comment]
@aldys33 I have some usage in my personal website, though it's pretty basic. Most of the complex usage is part of projects at work.

Here is an example with collection nesting.

https://github.com/mskelton/mskelton.dev/blob/635766f1db4d0e49ff8b9bae1c70c6310d200ba7/e2e/collections/BytesPage.ts#L14-L20

The idea behind nesting is things like modals or shared components where given a root selector, your nested collection is scoped to that, so if you have two tables on a page for example, you define a generic table collection, and then by nesting it under a given selector you can reuse the `Table` collection in both places with different roots. Things get really fun when you have multi-level nesting (e.g. form field inside a popover inside a table which is inside the app 🤯).
